pkg/tfgen/docs.go

```diff
@@ -216,6 +216,17 @@ func getMarkdownDetails(org string, provider string, resourcePrefix string, kind
 	return markdownBytes, markdownFileName, true
 }
 
+func (k DocKind) string() string {
```

Comment:
Can we name this function something that's not a golang keyword? like `asString` or `toString`?